.circleci/config.yml

```diff
@@ -178,6 +178,15 @@ jobs:
       - run: bundle exec i18n-tasks missing -t plural
       - run: bundle exec i18n-tasks check-consistent-interpolations
 
+  build-docker-image:
+    <<: *defaults
+    docker:
+    machine: true
+    steps:
+      - *attach_workspace
+      - run: docker build -t tootsuite/mastodon:latest .
+      - run: docker images
```

Comment:
Why run this line?

Reply:
Kind of habit and you can compare the image size with previous builds. [Right now it is 1.46GB](https://circleci.com/gh/tootsuite/mastodon/59636) which I think is quite a lot and we can compare if my other improvements not only decrease build time a lot but also image size.